server.rb

```diff
@@ -1,57 +1,74 @@
 require 'webrick'
+require 'webrick/https'
 require 'json'
 
 # default port to 3000 or overwrite with PORT variable by running
 # $ PORT=3001 ruby server.rb
-port = ENV['PORT'] ? ENV['PORT'].to_i : 3004
+port = ENV['PORT'] ? ENV['PORT'].to_i : 443
```

Comment:
may or may not want to change this back to 3004 as well. If anyone else wants to pull this and run it locally 443 will require running as sudo and you'd need to be passing env variables in your sudoers files.